test/common/schemaform/SaveInProgressErrorPage.unit.spec.jsx

```diff
@@ -70,7 +70,7 @@ describe('<SaveInProgressErrorPage>', () => {
     );
     const findDOM = findDOMNode(tree);
 
-    expect(findDOM.querySelector('.usa-alert').textContent).to.contain('We’re sorry, but something went wrong.');
+    expect(findDOM.querySelector('.usa-alert').textContent).to.contain('We’re sorry, but something went wrong. We can’t find your application');
```

Comment:
Should this text match the updated text above?

Reply:
It's a different message, but I'll make it more in line with the other ones.